### ISSUE TITLE
psalm forbids using void as a return type hint

### ISSUE THREAD

**Issue by greg0ire** [vimeo/psalm#3019]
I'm trying to setup psalm in https://github.com/doctrine/sql-formatter/pull/10 , but am blocked by the following error:

```
ERROR: ReservedWord - src/SqlFormatter.php:856:38 - void is a reserved word (see https://psalm.dev/095)
    private static function init() : void
```

The code can be found here:

https://github.com/doctrine/sql-formatter/blob/1b27af256e2f813c39a5fe27650f333b39e053b4/src/SqlFormatter.php#L855

The error message can be witnessed here: https://github.com/doctrine/sql-formatter/pull/10/checks?check_run_id=534231804#step:4:76

What's wrong?

**Comment by muglug** [comment]
It's in composer!

```
    "require": {
        "php": ">=5.2.4"
    },
```

I think that needs to be at least PHP 7.1 if you want to use void return types!

**Comment by greg0ire** [comment]
IIRC I tried that already, let me try again.

**Comment by greg0ire** [comment]
Ok now it works, I feel stupid! Anyway, thanks, looking forward to using Psalm in a CI pipeline for the first time!

**Comment by ktomk** [comment]
It's also possible to hint the php version explicitly like `--php-version=7.1`; and while trouble-shooting make use of `--no-cache` as well to prevent having anything cached between PHP version switches.
